std/experimental/ndslice/selection.d

```diff
@@ -747,36 +747,172 @@ pure nothrow unittest
               [[12, 13, 14], [17, 18, 19], [22, 23, 24]]]]);
 }
 
+deprecated("use:
+    Slice!(M, Range)
+    reshape
+        (size_t N, Range, size_t M)
+        (Slice!(N, Range) slice, size_t[M] lengths, ref int err)")
+Slice!(M, Range)
+    reshape
```

Comment:
This alignment is unusual, we have the function name flush with the returned type.